app/assets/javascripts/components/overview/course_stats.jsx

```diff
@@ -2,9 +2,23 @@ import React from 'react';
 import CourseUtils from '../../utils/course_utils.js';
 
 const CourseStats = ({ course }) => {
+  let viewData;
   if (course.upload_usages_count === undefined) {
     return <div className="stat-display"></div>;
   }
+  if (course.view_count === '0') {
+    viewData = (
+      <div className="stat-display__data">
+        {I18n.t('metrics.view_data_unavailable')}
+      </div>
+    );
+  } else {
+    viewData = (
+      <div className="stat-display__value">
+        course.view_count
```

Comment:
Needs braces, or it gets treated as a string literal.

One of the feature specs caught this, but this would be a good opportunity to add a javascript component test.